Readme.md

```diff
@@ -225,12 +225,45 @@ Upon being invoked, `Frisbee` returns an object with the following chainable met
         * `api.options(path, options)` - OPTIONS (*does not currently work - see tests*)
         * `api.patch(path, options)` - PATCH
 
-* Note that you can chain the `auth` method and an HTTP method together:
+    * Note that you can chain the `auth` method and an HTTP method together:
```

Comment:
Might want to change this to "and a".

Reply:
i think this might be correct 
https://www.quora.com/A-http-or-an-http-which-is-the-correct-grammatical-usage

Reply:
I'm going off of the fact it's a single HTTP method so "a" as in one. It's not a big issue though.